neqo-http3/src/headers_checks.rs

```diff
@@ -5,15 +5,19 @@
 // except according to those terms.
 
 use crate::{Error, MessageType, Res};
+use enumset::{enum_set, EnumSet, EnumSetType};
 use neqo_common::Header;
 
-const PSEUDO_HEADER_STATUS: u8 = 0x1;
-const PSEUDO_HEADER_METHOD: u8 = 0x2;
-const PSEUDO_HEADER_SCHEME: u8 = 0x4;
-const PSEUDO_HEADER_AUTHORITY: u8 = 0x8;
-const PSEUDO_HEADER_PATH: u8 = 0x10;
-const PSEUDO_HEADER_PROTOCOL: u8 = 0x20;
-const REGULAR_HEADER: u8 = 0x80;
+#[derive(EnumSetType, Debug)]
+enum PseudoHeaderState {
+    Status,
+    Method,
+    Scheme,
+    Authority,
+    Path,
+    Protocol,
+    None,
```

Comment:
Try

```suggestion
    Regular,
```

Or something like that.